pipelines/optimus/README.md

```diff
@@ -1,33 +1,223 @@
+* Reference genome fasta file: genome fasta file (must match star reference and organism)
+
+### V2 or V3 Chemistry Input 
+
+The chemistry (V2 or V3) for the sequencing experiment needs to be specified in the [optimus.wdl code](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Optimus.wdl). The default is set to V2 using the following the command:
+
+String chemistry = "tenX_v2" 
+
+To change to V3 chemistry, use "tenX_v3". 
+
+
+# Running Optimus
+
 * [Optimus.wdl](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Optimus.wdl) in the pipelines/optimus folder,  of the repository, implements the workflow by importing individual tasks in task based WDLs in skylab/library.
 
-#### Inputs
-Example input file locations are specified in a json file, e.g., [here](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/example_test_inputs.json). The following table provides information on specific input values.
 
+## Optimus Modules Summary
+
+Here we describe the modules of Optimus; [the code](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Optimus.wdl) and [library of tasks](https://github.com/HumanCellAtlas/skylab/tree/master/library/tasks) are available through Github.
+
+Overall, the workflow:
+1. Converts R2 Fastq file to BAM
+2. Corrects and attaches 10X Barcodes 
+3. Aligns reads to the genome
+4. Annotates genes with aligned reads
+5. Generates an expression count matrix in a UMI-aware fashion
+6. Detects empty droplets
+7. Calculates summary statistics (metrics) and count
+8. Produces a count matrix
+9. Returns output in BAM, Zarr, or loom file formats
+
+Special care is taken to flag but avoid the removal of reads that are not aligned or that do not contain recognizable barcodes. This design (which differs from many pipelines currently available) allows use of the entire dataset by those who may want to use alternative filtering or leverage the data for methodological development associated with the data processing.
+
+### 1. Converting R2 Fastq file to BAM
+
+Because the pipeline processing steps require a BAM file format, the first step of Optimus is to [convert](https://broadinstitute.github.io/picard/command-line-overview.html#FastqToSam) the R2 Fastq file, containing the alignable genomic information, to a BAM file.
+
+### 2. Correcting and Attaching Cell Barcodes
+
+Although the function of the cell barcodes is to identify unique cells, barcode errors can arise during sequencing (such as incorporation of the barcode into contaminating DNA or sequencing and PCR errors), making it difficult to distinguish unique cells from artifactual appearances of the barcode. Barcode errors are evaluated in the [Attach10xBarcodes](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/Attach10xBarcodes.wdl), which compares the sequences against a whitelist of known barcode sequences.
```

Comment:
I like the idea of adding specific Improvements doc and contributions doc since these are items that might change at a different pace the pipeline itself. I could reference the two separate docs in this main ReadMe so people know where to go.